src/canvas/Renderer.tsx

```diff
@@ -75,6 +82,8 @@ class Renderer implements IRenderer {
         }
         this._isTracing = false;  
         this._currPath = null;
+
+        this._canvas.dispatchEvent(new Event('endTracing'));
```

Comment:
this should only fire if this._isTracing was true before line 83. Otherwise the review page pops up even if the mouse just gets in and out of the canvas without ever clicking

Reply:
This should be fixed now too :)